cimr/processor/process.py

```diff
@@ -340,54 +318,54 @@ def make_composite_id(self):
 
         if set(VAR_COMPONENTS).issubset(set(self.included_header)):
             logging.debug(f' checking {VAR_COMPONENTS} for missing values.')
-            self.summary_data.dropna(
+            self.data.dropna(
                 subset=VAR_COMPONENTS, inplace=True
             )
             logging.debug(f' rows with missing {VAR_COMPONENTS} are dropped.')
 
             logging.debug(f' making variant_id using {VAR_COMPONENTS}.')
-            self.summary_data['chrom'] = self.summary_data['variant_chrom']
+            self.data['chrom'] = self.data['variant_chrom']
             self.check_chrom()
             logging.info(f' verifying variant positions are int values.')
-            self.make_int('variant_pos')
+            self.data = make_int(self.data, 'variant_pos')
             logging.debug(f' changing verified values to str.')
-            self.make_str('variant_pos')
+            self.data = make_str(self.data, 'variant_pos')
 
-            self.summary_data['pos'] = self.summary_data['variant_pos']
-            self.summary_data['ref'] = self.summary_data['non_effect_allele']
-            self.summary_data['alt'] = self.summary_data['effect_allele']
-            self.summary_data['build'] = self.genome_build
+            self.data['pos'] = self.data['variant_pos']
+            self.data['ref'] = self.data['non_effect_allele']
+            self.data['alt'] = self.data['effect_allele']
+            self.data['build'] = self.genome_build
 
             # make variant_id from components
             self.make_variant_id()
             # recall included_header from columns including variant_id
             self.included_header = intersect_set(
-                HEADER, self.summary_data.columns
+                HEADER, self.data.columns
             )
 
         else:
             logging.error(f' missing columns necessary to make variant_id.')
             sys.exit(1)
 
 
-    def check_file(self, summary_data):
+    def check_file(self, data):
```

Comment:
What type is `data` here? This should be in the docstring. It looks like a dataframe, but the function name is `check_file`

Reply:
`check_data` is probably a better name. By the way, @ypar , do you think it is a good idea to initialize `self.data` in `self.__init__`?